Perlang.sln

```diff
@@ -49,4 +53,13 @@ Global
 		{CD78DA11-CB31-4892-8F96-3B7FD7015B83}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{CD78DA11-CB31-4892-8F96-3B7FD7015B83}.Release|Any CPU.Build.0 = Release|Any CPU
 	EndGlobalSection
+	GlobalSection(NestedProjects) = preSolution
+		{B338D29B-A85D-4650-9BFF-FBFE55258C58} = {B924E83B-6AFE-4ABF-B79D-444F92582892}
+		{EC950B39-0187-41BD-A9AA-FE18F73BD30F} = {B924E83B-6AFE-4ABF-B79D-444F92582892}
+		{2E39F005-52FF-404D-B570-0AB2E9081272} = {B924E83B-6AFE-4ABF-B79D-444F92582892}
+		{F1F9229C-D764-4A38-BEA2-1F86A1C2FF1A} = {B924E83B-6AFE-4ABF-B79D-444F92582892}
+		{20267530-E991-4936-BFDC-0C5A1EDE90E7} = {B924E83B-6AFE-4ABF-B79D-444F92582892}
+		{CD78DA11-CB31-4892-8F96-3B7FD7015B83} = {F13F8E4E-5231-4321-9722-1BFCC08E5BB5}
+		{1874E267-5D6F-4EA4-8938-F2A0D19F27B1} = {F13F8E4E-5231-4321-9722-1BFCC08E5BB5}
+	EndGlobalSection
```

Comment:
Drive-by-change, just making the Solution view in Rider/Visual Studio a bit more readable.